addons/l10n_si/data/template/account.tax-si.csv

```diff
@@ -1,193 +1,257 @@
+"","","","","","","","","","","100.00","base","refund","-31a||-ir_25c||-pr_9","",""
+"","","","","","","","","","","100.00","tax","refund","-42||-pr_19","gd_acc_160470",""
+"","","","","","","","","","","-100.00","tax","refund","+25a||+ir_21a","gd_acc_260470",""
+"gd_taxp_st_2","100","22%","22% input VAT - self-assessment Article 76a","22% Art. 76a","22.00","percent","purchase","tax_group_so_22","","100.00","base","invoice","+31a||+ir_25c||+pr_9","","Vstopni DDV 22% - samoobdavčitev 76.a člen"
+"","","","","","","","","","","100.00","tax","invoice","+41||+pr_18","gd_acc_160460",""
+"","","","","","","","","","","-100.00","tax","invoice","-25||-ir_20","gd_acc_260460",""
+"","","","","","","","","","","100.00","base","refund","-31a||-ir_25c||-pr_9","",""
+"","","","","","","","","","","100.00","tax","refund","-41||-pr_18","gd_acc_160460",""
+"","","","","","","","","","","-100.00","tax","refund","+25||+ir_20","gd_acc_260460",""
+"gd_taxe_out_0_no_no_1","110","0%","0% exit VAT - other exemptions","0% O Exempt","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+11||+ir_7","","Izstopni DDV 0% - druge oprostitve"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-11||-ir_7","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_no_no_2","110","0%","0% output VAT - does not go on the books","0% Not in books","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+11||+ir_7","","Izstopni DDV 0% - ne gre v knjigo"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-11||-ir_7","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_1","130","0%","Exit VAT 0% EU - Goods","0% EU G","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_10a||+rp_a3","","Izstopni DDV 0% EU - blago"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-12||-ir_10a||-rp_a3","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_2","140","0%","0% EU exit VAT - customs procedure Articles 42 and 43 - goods","0% EU G Art. 42-43","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_10b||+rp_a4","","Izstopni DDV 0% EU - carinski postopek 42. in 43. člen - blago"
+"","","","","","","","","","","100.00","tax","invoice","","",""
+"","","","","","","","","","","100.00","base","refund","-12||-ir_10b||-rp_a4","",""
+"","","","","","","","","","","100.00","tax","refund","","",""
+"gd_taxe_out_0_eu_goods_3","150","0%","0% EU output VAT - triangular supplies - goods","0% EU G Tri","0.00","percent","sale","tax_group_0","","100.00","base","invoice","+12||+ir_11||+rp_a5","","Izstopni DDV 0% EU - tristranske dobave - blago"

```

Comment:
Does `output` mean `export` here?

Reply:
Output means sale i think

Reply:
That's what i put in the name yes :smile: i think the description can stay like that

Reply:
Ok for the name, but I would really remove "output" from the descrpition, or change it to "sale". It's unnecessary and confusing.